src/components/ValidationPrediction.tsx

```diff
@@ -21,26 +21,43 @@ export default function ValidationPrediction(props: Props) {
 
   const selectedRegionTitle = selectedRegion ? getRegionTitle(selectedRegion) : "";
 
+  const filteredRegions = Object.entries(props.options.selectedRegions).filter(([isSelected]) => isSelected);
```

Comment:
If you're going to use `Object.entries`, the predicate should be `([key, isSelected]) => isSelected`. I suggest to use `Object.keys`

```suggestion
  const filteredRegions = Object.keys(props.options.selectedRegions).filter((key) => props.options.selectedRegions[key]);
```

Reply:
I will fix this